Source/include/Ably/ARTChannelOptions.h

```diff
@@ -8,6 +8,7 @@ NS_ASSUME_NONNULL_BEGIN
 /**
  * Passes additional properties to an `ARTRestChannel` object, such as encryption.
  */
+NS_SWIFT_SENDABLE
```

Comment:
Why have you marked the mutable options / query types as `NS_SWIFT_SENDABLE`? They’re not sendable; they don't synchronise access to their mutable state and aren't safe to share between isolation domains. You didn't mark `ARTChannelOptions` as sendable in #1963; why would these types be different?

I think it would be helpful for you to explain in more detail what was the motivation for the current PR, so that we can figure out how to solve it. Marking these types (at least in their current state) as sendable isn't correct.